src/js/tabs/configuration.js

```diff
@@ -511,11 +511,21 @@ TABS.configuration.initialize = function (callback, scrollPosition) {
                 self.analyticsChanges['BidirectionalDshot'] = newValue;
 
                 MOTOR_CONFIG.use_dshot_telemetry = value;
+				
             });
 
             $('input[name="motorPoles"]').val(MOTOR_CONFIG.motor_poles);
         }
 
+        function hidemotorPoles() {
+            if ($("input[id='dshotBidir']").is(':checked') ||  $("input[name='ESC_SENSOR']").is(':checked')){
+                $('div.motorPoles').show();
+
+            } else {
+                $('div.motorPoles').hide();
+            }
+		}
```

Comment:
Caution with the indentation 😉

I suggest better something like:
```
let motorPolesVisible = $("input[id='dshotBidir']").is(':checked') ||  $("input[name='ESC_SENSOR']").is(':checked');
$('div.motorPoles').toggle(motorPolesVisible);
```
The number of conditions can grow in future versions and this is cleaner.